src/phoenix/server/api/routers/v1/__init__.py

```diff
@@ -11,4 +12,5 @@
     Route("/v1/datasets/upload", datasets.post_datasets_upload, methods=["POST"]),
```

Comment:
Unrelated to this PR, but do we need the `upload` here?

Reply:
what do you mean?

Reply:
As a suffix on the route. Wondering whether it makes sense to just POST to `/v1/datasets`.

Reply:
ah yeah, I'd probably agree, but let's resolve this separately